[TW-714] Doc user shared auth, OAuth for on-prem, and multiple APIs to one repo

mark-anthony-postman opened the pull request:
- "Using a remote Git repository" page - Updated connecting to cloud and on-prem repos, added section about connecting multiple APIs to a repo
- "Setting up Postman" page - Updated "Connected accounts" section
- "Using installed apps in integrations" page - Added GitHub Enterprise and GitLab Self-Managed

MatthewPinsker-postman reviewed (approved):
LGTM!

MatthewPinsker-postman commented on src/pages/docs/designing-and-developing-your-api/versioning-an-api/using-external-git-repo.md:
```diff
@@ -82,49 +84,134 @@ You can connect to a GitHub, Bitbucket, GitLab, or Azure DevOps repository hoste
+
+#### Creating an installed app
+
+[Installed apps](/docs/integrations/installed-apps/) streamline the process of adding integrations by enabling all members of a team to use the same stored authorization details. To create an installed app for GitHub Enterprise Server or GitLab Self-Managed, do the following:
```
According to the [style guide](https://postmanlabs.atlassian.net/wiki/spaces/TW/pages/3978756522/Procedures#Lead-in-sentences), we should put lead-in sentences on their own line.

mark-anthony-postman replied:
Great, thanks for spotting this!

MatthewPinsker-postman commented on src/pages/docs/designing-and-developing-your-api/versioning-an-api/using-external-git-repo.md:
```diff
@@ -82,49 +84,134 @@ You can connect to a GitHub, Bitbucket, GitLab, or Azure DevOps repository hoste
+
+#### Connecting an API using an installed app
+
+After you [create an installed app](#creating-an-installed-app), any API editor on the team can use it to connect an API to GitHub Enterprise Server or GitLab Self-Managed. To connect an API using an installed app, do the following:
```
According to the [style guide](https://postmanlabs.atlassian.net/wiki/spaces/TW/pages/3978756522/Procedures#Lead-in-sentences), we should put lead-in sentences on their own line.

jatin3893 commented:
<img width="735" alt="Screenshot 2023-03-08 at 10 25 48 PM" src="https://user-images.githubusercontent.com/3127114/223778139-39c49ff7-f90f-4ae0-84ef-ef21c76048c9.png">

Sharing feedback as shared on Slack^

jatin3893 commented:
Do we need to specify any details like "This is supported from Postman v10.12 onwards" or something like that?

mark-anthony-postman commented:
> Do we need to specify any details like "This is supported from Postman v10.12 onwards" or something like that?

@jatin3893 We generally don't do this, as the Learning Center docs are always for the latest version of Postman

jkonrath-postman commented on src/pages/docs/designing-and-developing-your-api/versioning-an-api/using-external-git-repo.md:
```diff
@@ -82,49 +84,138 @@ You can connect to a GitHub, Bitbucket, GitLab, or Azure DevOps repository hoste
+    * `/postman/collections` - Collections for the `notifications-v2` API
 
-Postman securely stores your personal access token. After connecting to an on-premises repository, you can connect to other repositories using the same domain URL without having to re-enter your token. When connecting to a repository, select **Change** under **Authenticated Domain** if you want to use a different domain URL.
+You must use a unique branch + folder combination for each API you [connect to the repository](#connecting-to-a-repository). For example, if you connect one API to the `/postman/schemas` folder on the `notifications-v1` branch, you can't connect another API to the same folder on the same branch. You must use a different branch or a different schema folder.
```
```suggestion
You must use a unique branch and folder combination for each API you [connect to the repository](#connecting-to-a-repository). For example, if you connect one API to the `/postman/schemas` folder on the `notifications-v1` branch, you can't connect another API to the same folder on the same branch. You must use a different branch or a different schema folder.
```

jkonrath-postman reviewed (approved):
LGTM!

mark-anthony-postman commented:
> Sharing feedback (same as the one on slack)

@jatin3893 Thanks for the great feedback! I've added your suggestions to the documentation and updated the PR.

shashankawasthi88 commented on src/pages/docs/designing-and-developing-your-api/versioning-an-api/using-external-git-repo.md:
```diff
@@ -79,52 +81,141 @@ You can connect to a GitHub, Bitbucket, GitLab, or Azure DevOps repository hoste
+1. Create an OAuth app by following the instructions for your Git provider ([GitHub Enterprise Server](https://go.pstmn.io/github-oauth-app-setup) or [GitLab Self-Managed](https://go.pstmn.io/GitLab-OAuth-app-setup)).
+
+    * For **Homepage URL** enter: `https://www.postman.com`
+    * For **Authorization callback URL** or **Redirect URI**, enter the authorization callback URL provided in Postman.
```
Should we provide the URL here as well?

mark-anthony-postman replied:
@shashankawasthi88 Sure, that makes sense as long as the callback URL is always the same.

shashankawasthi88 reviewed (approved):
@mark-anthony-postman this looks good, but we should add the call back URL in docs as well/
